Do not strip mingw libraries from list of Fortran standard linker flags.

diff -urN autoconf-2.72/lib/autoconf/fortran.m4.orig autoconf-2.72/lib/autoconf/fortran.m4
--- autoconf-2.72/lib/autoconf/fortran.m4.orig	2024-10-30 13:31:24.456674400 +0100
+++ autoconf-2.72/lib/autoconf/fortran.m4	2024-12-11 16:08:46.415095400 +0100
@@ -705,7 +705,7 @@
 	-lang* | -lcrt*.o | -lc | -lgcc* | -lSystem | -libmil | -little \
 	  |-LANG:=* | -LIST:* | -LNO:* | -link)
 	  ;;
-	-lkernel32 | -lmingw* | -lmoldname)
+	-lkernel32)
 	  # Ignore this library only on Windows-like systems.
 	  case $host_os in
 	  cygwin* | msys* | mingw* | windows*) ;;
